packages/create-cloudflare/templates-experimental/vue/js/server/index.js

```diff
@@ -0,0 +1,13 @@
+export default {
+	fetch(request, env) {
+	  const url = new URL(request.url);
```

Comment:
this line, alongside others in this file, is mixing tabs and spaces, I think that's not intentional is it?